bindings/rust/src/lib.rs

```diff
@@ -131,6 +131,84 @@ impl From<f64> for Value {
     }
 }
 
+/// A WebAssembly value i32/i64/f32/f64 with its type specified.
+pub enum TypedValue {
+    U32(u32),
```

Comment:
Since `sys::FizzyValueType` only supports an "unsigned" type for 32/64-bit integers, we are matching that here. The naming discrepancy is that I wanted to make clear U32/U64 matches the unsigned Rust type `u32`/`u64`.